bitwarden_license/src/Commercial.Core/SecretsManager/Commands/Projects/DeleteProjectCommand.cs

```diff
@@ -72,8 +75,11 @@ public DeleteProjectCommand(IProjectRepository projectRepository, ICurrentContex
 
         if (deleteIds.Count > 0)
         {
+            var secretIds = results.SelectMany(projTuple => projTuple.Item1?.Secrets?.Select(s => s.Id) ?? Array.Empty<Guid>()).ToList();
+            await _updateSecretCommand.UpdateRevisionDates(secretIds);
```

Comment:
Normally, with the CQRS pattern, we want to avoid calling commands within commands. 

What do you think about just calling the `_secretRepository.UpdateRevisionDates(secretIds);` here?

Reply:
Thanks for pointing that out! Sounds good, updated 👍